lib/helper.js

```diff
@@ -0,0 +1,12 @@
+var semver = require('semver');
+
+exports.DEFAULT_LB_VERSION = '2.x';
+
+/**
+ * validate loopback version
+ * @param version
+ * @returns valid loopback version or default if invalid
+ */
+exports.validateLoopbackVersion = function(version) {
+  return semver.validRange(version) == null ? this.DEFAULT_LB_VERSION : version;
```

Comment:
Using `this.DEFAULT_LB_VERSION` means the following code won't work:

```js
var validateLoopbackVersion = require('../lib/helper').validateLoopbackVersion;
validateLoopbackVersion(...);
```

I am proposing to use `exports.DEFAULT_LB_VERSION` or alternatively save the constant to a top-level variable.

```
var DEFAULT_LB_VERSION = '2.x';
exports.DEFAULT_LB_VERSION = DEFAULT_LB_VERSION;
//...
```